src/components/EditServerDetails.js

```diff
@@ -178,8 +197,7 @@ class EditServerDetails extends Component {
           <div className='input-with-button'>
             <ListDropdown name={this.props.name} value={this.data[name]} moreClass={'has-button'}
               optionList={list} emptyOption={emptyOptProps} selectAction={handler}/>
-            <ActionButton type={'default'} clickAction={addAction} moreClass={'inline-button'}
-              displayLabel={translate(buttonLabel) + ' ...'}/>
+            {this.props.mode === undefined && this.renderButtonForDropDown(addAction, buttonLabel)}
```

Comment:
maybe with this check for this.props.mode all over, it should be moved to its own function with a descriptive name... like isPreInstall or isInstalling or something... and just call that function everywhere that `this.props.mode === undefined`  is essentially checking if its going on during install or after

Reply:
this.props.mode means it is not in any addserver mode which day0 mode. yeah...it is all over the place , ok, will think about something  and update.

Reply:
ah ok, I see, since its not a change, we can worry about the translation feedback some other time (it likely wont come up)

Reply:
I just dont think its obvious to a casual observer what this check is doing... doing exactly the same thing inside another function that has a name would make it more obvious.... for example, in this case the code would read:
`{this.isInitialInstall() && this.renderButtonForDropDown(addAction, buttonLabel)}`  ... although hopefully someone can come up with a better function name than isInitialInstall  ;)  ... could reverse it and do `{!this.isPostInstall() && ...}`

Reply:
ok will update call it  isInstall for now unless someone comes up with a better name